DuckDuckGo/Subscription/ViewModel/SubscriptionSettingsViewModel.swift

```diff
@@ -76,6 +79,14 @@ final class SubscriptionSettingsViewModel: ObservableObject {
         }
     }
     
+    private func setupNotificationObservers() {
+        signOutObserver = NotificationCenter.default.addObserver(forName: .accountDidSignOut, object: nil, queue: .main) { [weak self] _ in
```

Comment:
You need to also remove closure based observers e.g. on deinit

Reply:
oh, yeah.  Missed it.  Good call.